docs/application/dotnet/guides/account/data-sync.md

```diff
@@ -138,9 +138,9 @@ You can add user data to a sync job as an account information instance or as a d
 <a name="on_demand_sync"></a>
 ## Requesting an On-demand Sync Job
```

Comment:
## Request on-demand sync Job

Reply:
Done. 'Request On-demand Sync Job' was applied. Without applying 'Sentence case' - according to our approach https://github.com/Samsung/tizen-docs/pull/1387#discussion_r575045312